frontend/components/forms/RegistrationForm/_styles.scss

```diff
@@ -33,25 +33,25 @@
 
     &--admin {
       left: 0;
-      top: unquote("max(56%, 560px)");
+      top: unquote("max(60%, 625px)");
```

Comment:
Do we need to wrap these in the unquote function? Is `top: max(60%, 625px);` not valid on its own? Maybe there's some cross-browser compatibility thing that makes the unquote necessary. I'm just curious if you happen to know. No need to change anything in the PR.